modules/cluster/irsa.tf

```diff
@@ -153,10 +153,10 @@ module "iam_assumable_role_cert_manager" {
   source                        = "terraform-aws-modules/iam/aws//modules/iam-assumable-role-with-oidc"
   version                       = "~> v2.13.0"
   create_role                   = true
-  role_name                     = substr("tf-${var.cluster_name}-sa-role-cert_manager-${local.generated_seed}", 0, 60)
+  role_name                     = var.is_jx2 ? substr("tf-${var.cluster_name}-sa-role-cert_manager-${local.generated_seed}", 0, 60) : substr("${var.cluster_name}-cert-manager-cert-manager", 0, 60)
```

Comment:
same, dont need substr.

Reply:
Switched truncation to happen per-cluster and per-namespace